packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -121,33 +124,63 @@ class ExpansionPanelList extends StatelessWidget {
   /// The duration of the expansion animation.
   final Duration animationDuration;
 
+  /// Whether the Expansion Panel List can have multiple panels open
+  /// simultaneously or at most one.
+  ///
+  /// If set to false and a closed panel is opened, all other opened panels will
+  /// be closed. In this state there can be exactly one or zero panels open.
+
+  final bool allowMultiplePanelsOpen;
+
+  @override
+  State<StatefulWidget> createState() => new _ExpansionPanelListState();
+}
+
+class _ExpansionPanelListState extends State<ExpansionPanelList> {
+
+  /// Creates an expansion panel list widget. The [expansionCallback] is
+  /// triggered when an expansion panel expand/collapse button is pushed.
+  List<bool> widgetOpenValue;
+
+  @override
+  void initState() {
+    super.initState();
+    widgetOpenValue = new List<bool>(widget.children.length);
+
+    for(int i = 0; i < widget.children.length; i++)
+      widgetOpenValue[i] = false;
```

Comment:
this won't work. For example, what if you have three children A, B, and C, and B is opened, and then A is removed.

Reply:
You're right, I didn't realize I needed to support removal and addition of items. I'll add that functionality.